src/main/resources/org/codefirst/SimpleThemeDecorator/global.groovy

```diff
@@ -4,7 +4,7 @@ import lib.FormTagLib
 
 def f = namespace(FormTagLib)
 
-f.section(title: _("Theme")) {
+f.section(title: _("Customizable theme")) {
```

Comment:
I changed this as having it next to the theme manager plugin looked strange with basically the same heading, not wedded to the wording but it matched the customizeable header just after it:

<img width="1263" alt="image" src="https://github.com/jenkinsci/simple-theme-plugin/assets/21194782/07738591-2212-43d8-97f7-d79e438387fb">

<img width="1192" alt="image" src="https://github.com/jenkinsci/simple-theme-plugin/assets/21194782/c6736a87-7cb0-45ef-8e42-fb7a7beeccc6">

Reply:
Well, I have no better idea, so let's go with that 😄